detection_rules/packaging.py

```diff
@@ -460,6 +471,38 @@ def generate_xslx(self, path):
         doc.populate()
         doc.close()
 
+    def _generate_registry_package(self, save_dir):
+        """Generate the artifact for the oob package-storage."""
+        from .schemas.registry_package import RegistryPackageManifest
+
+        assert self.registry_data
+
+        manifest = RegistryPackageManifest.from_dict(self.registry_data)
```

Comment:
I think it would make more sense to give this class an already assembled RegistryPackageManifest, but we can do this later with a restructur